ion-hash/Cargo.toml

```diff
@@ -22,7 +22,5 @@ edition = "2018"
 [dependencies]
 ion-rs = { path = "../", version = "0.5" }
 ion-c-sys = { path = "../ion-c-sys", version = "0.4" }
+digest = "0.9.0"
 sha2 = { version = "0.9.3" }
```

Comment:
I'm preparing to make it an optional dependency so that consumers don't need digest implementations they're not using. I had it as optional, but I had some editor issues, .. For now, I'll just have the version verbatim.